source/_patterns/03-organisms/gallery/gallery-overlay.json

```diff
@@ -4,23 +4,22 @@
   "slides": [
     {
       "slide_title": "Mercado Little Spain",
-      "break_m": "700",
       "img": {
-        "src_s": "https://picsum.photos/400/700?image=122"
+        "src_s": "https://cmsplatypus.gothamist.com/gothamistgallery/2019/3/15/df594e893hyfood-14-jpg-mobile.jpeg"
```

Comment:
just a heads up, we'll probably be turning off `cmsplatypus.gothamist.com` some time in the near future so you might want to use a different image here and elsewhere where you're pointing to this server...

Reply:
Gotcha!